spec/views/hyrax/admin/collection_types/index.html.erb_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+RSpec.describe 'hyrax/admin/collection_types/index.html.erb', type: :view do
+  before do
+    assign(:collection_types, [
+             FactoryGirl.create(:collection_type, title: 'Test Title 1'),
+             FactoryGirl.create(:collection_type, title: 'Test Title 2')
```

Comment:
The failing test...
```
Failure/Error: FactoryGirl.create(:collection_type, title: 'Test Title 1'),
     ActiveRecord::RecordInvalid:
       Validation failed: Machine can't be blank
```
is happening because machine_id is required.  You can fix by adding `machine_id: "test_title_1"` to your factory.  See [predefined user_collection factory](https://github.com/samvera/hyrax/blob/collections-sprint/spec/factories/collection_types.rb#L3-L6) as an example.

Also, will the test run correctly with `build` instead of `create`?  `build` is faster than `create` and always preferable unless saving to the DB is required for the test.